docs/developers_guide/FlinkComponents.md

```diff
@@ -1,22 +1,72 @@
 # Flink components
- 
-Sources, sinks and custom transformations are based on 
-Flink API. Access to various helpers is provided by 
-`FlinkCustomNodeContext`. Special care should be taken to handle
-- lifecycle
-- exception handling properly
+
+Sources, sinks and custom transformations are based on
```

Comment:
Is this term _custom transformations_ coming from Flink's vocabulary or is it (disused by now) Nu lingo? If from Flink - than OK, if not, I would just write _other Flink based(?) components_

Reply:
https://docs.nussknacker.io/documentation/docs/developers_guide/Basics -> we do have separate pages for common concepts